go-controller/hack/test-go.sh

```diff
@@ -53,7 +53,7 @@ function testrun {
     fi
     if grep -q -r "ginkgo" ."${path}"; then
 	    prefix=$(echo "${path}" | cut -c 2- | sed 's,/,_,g')
-        ginkgoargs="-ginkgo.v -ginkgo.reportFile ${TEST_REPORT_DIR}/junit-${prefix}.xml"
+        ginkgoargs="-ginkgo.v -ginkgo.reportFile ${TEST_REPORT_DIR}/junit-${prefix}.xml -ginkgo.flakeAttempts=3"
```

Comment:
let's not have this commit in this PR. I'd rather those unit tests flake honestly and people fix them, but we can discuss it in another PR.